src/static/css/2019.css

```diff
@@ -1304,12 +1310,28 @@ p.copyright a {
   transition: all 200ms linear;
 }
 
-.nav-dropdown-btn.dropdown-open::after {
+.nav-dropdown-btn.dropdown-open::after,
+.nav-dropdown-list-current::after {
   width: 17px;
   right: 16px;
   margin-top: -9px;
 }
 
+.table-of-contents .nav-dropdown-btn.dropdown-open::after {
+  height: 7px;
+  width: 7px;
+  margin-top: -2px;
+  right: 23px;
+  -webkit-transform: rotate(-45deg);
+  -ms-transform: rotate(-45deg);
+  transform: rotate(-45deg);
```

Comment:
BTW do we support `prefers-reduced-motion` yet? If/When we do, it's worth disabling the transition on this property.

Reply:
We do in JavaScript:

https://github.com/HTTPArchive/almanac.httparchive.org/blob/e68274c79677de4a298abc0c6e3dae859ed16f52/src/static/js/chapter.js#L280-L287

Added a CSS override in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1512/commits/65eca17ee0ef5f66198886baf6c2684a8da4dce5 as based on [this web.dev article](https://web.dev/prefers-reduced-motion/#(bonus)-forcing-reduced-motion-on-all-websites).